### PR TITLE
feat: add extra configuration for db replicas [BB-5273]

### PR DESCRIPTION
**Pull request by Cup0fCoffee**
### Description

Specify additional configuration for the db replicas instances, so that UQ's DaaS team can use Qlik Replicate tool with the replicas.

### References
[BB-5273](https://tasks.opencraft.com/browse/BB-5273)

### TODO
- [ ] Update branch for sql module in configuration secure for the client

### CODE REVIEW

File: modules/services/sql/rds.tf [hunk 1]
```diff
@@ -66,6 +67,26 @@ resource aws_db_instance mysql_rds_replicas {
...
+    value        = "CRC32"
+  }
+  parameter {
+    name         = "binlog_format"
+    value        = "row"
+  }
+  parameter {
+    name         = "binlog_row_image"
+    value        = "full"
+  }
```
Comment (tecoholic) [inline modules/services/sql/rds.tf:89]: @Cup0fCoffee Is this necessary to be explicitly defined here? All these values seem to be the default.

AWS Docs says

> RDS for MySQL parameters are set to the default values of the storage engine that you have selected. For more information about MySQL parameters, see the MySQL documentation

and MySQL docs:

* [binlog_checksum](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_checksum)
* [binlog_format](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_format)
* [binlog_row_image](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_format)

**Review by tecoholic — CHANGES_REQUESTED** [review]

**Comment by tecoholic** [comment]
@Cup0fCoffee I am requesting changes, which is to remove the db parameters as they seem to be all default values. If you think, the cost of defining the defaults in the Terraform would serve the benefit of working as a documentation for the client, consider this as a :+1: and ignore my request for changes.

- [x] I read through the code.

File: modules/services/sql/rds.tf (hunk 1 above)
Comment (Cup0fCoffee) [reply modules/services/sql/rds.tf:89]: @tecoholic
> Is this necessary to be explicitly defined here? All these values seem to be the default.

Yes. They are in fact default for the MySQL, but the defaults on AWS RDS are different. You can verify that by going into the client's console, find the master DB and in the configuration tab you will see that it's pointing to the `default.mysql5.7` parameters group. The defaults for these values are different there. You can also verify it by connecting directly to the dbs and comparing the values.

They are ok for the master DB, but we need them to be different for the replicas, hence we explicitly overwrite them. There are also docs about it on AWS, but I couldn't find them this time.

Comment (tecoholic) [reply modules/services/sql/rds.tf:89]: @Cup0fCoffee Ah.. Okay. I didn't somehow come across this difference in the docs. Thank you for clarifying.

**Pull request merged**